.github/workflows/build.yml

```diff
@@ -92,7 +95,6 @@ jobs:
       docker-username: ${{ github.actor }}
       version: ${{ needs.versions.outputs.version }}
       versionNumber: ${{ needs.versions.outputs.version }}
-      dockerContext: '.'
```

Comment:
note: I think it will be better if we leave this parameter as is. Even if the value is the same as the default one. Nothing stops somebody from changing the default value in reusable workflow